src/Pages/Stay/StayMain/Calendar/StayCalendar.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import 'react-dates/initialize';
+import './datepicker.css';
+import { DateRangePicker } from 'react-dates';
+
+import styled from 'styled-components';
+
+const StayCalendar = ({ onCreate }) => {
+  const [startDate, setStartDate] = useState(null);
+  const [endDate, setEndDate] = useState(null);
+  const [startDate2, setStartDate2] = useState(null);
+  const [endDate2, setEndDate2] = useState(null);
+  const [startPlaceholder] = useState(new Date());
+  const [endPlaceholder] = useState(new Date());
+  const [focusedInput, setFocusInput] = useState(null);
+
+  const onDatesChange = ({ startDate, endDate }) => {
+    setStartDate(startDate);
+    setEndDate(endDate);
+
+    if (startDate !== null && endDate !== null) {
+      setStartDate2(startDate.format('YYYY-MM-DD'));
+      setEndDate2(endDate.format('YYYY-MM-DD'));
+    }
+
+    onCreate({
+      start: startDate2,
+      end: endDate2,
+    });
+  };
+  console.log(startDate);
+  console.log(endDate);
+
+  const getFormatStartDate = date => {
+    let year = date.getFullYear();
+    let month = 1 + date.getMonth();
+    month = month >= 10 ? month : '0' + month;
+    let day = date.getDate();
+    day = day >= 10 ? day : '0' + day;
+    return year + '-' + month + '-' + day;
+  };
+  const getFormatEndDate = date => {
+    let year = date.getFullYear();
+    let month = 1 + date.getMonth();
+    month = month >= 10 ? month : '0' + month;
+    let day = 1 + date.getDate();
+    day = day >= 10 ? day : '0' + day;
+    return year + '-' + month + '-' + day;
+  };
+  return (
+    <SearchDateContainer>
+      <DateRangePicker
+        startDatePlaceholderText={getFormatStartDate(startPlaceholder)}
+        endDatePlaceholderText={getFormatEndDate(endPlaceholder)}
```

Comment:
위 코드 리뷰를 반영한다면, 아래와 같이 사용할 수 있을 것 같습니다.
```suggestion
        startDatePlaceholderText={formatDate(date, "startDate")}
        endDatePlaceholderText={formatDate(date, "endDate")}
```

Reply:
감사합니다 바로적용했습니다